### ISSUE TITLE
Explosive Death Destroys Grave, Removes Items

### ISSUE THREAD

**Issue by morphiussys** [twothe/twograve#2]
So it seem that when I (or those on the server I am part of) die, the grave is removing items when it spawns (so dying near machines n stuff is risky).  Also, when we die an explosive death, the grave seems to spawn too quickly and gets destroyed by the original explosion.

**Comment by twothe** [comment]
That is correct and known. The "search" algorithm currently just takes the block the player is standing in, and assumes that it is empty.

In a future version the mod should actually search for a valid location close-by.

**Comment by morphiussys** [comment]
Ok cool.
